app/components/billing-header/template.hbs

```diff
@@ -2,7 +2,7 @@
   <div class="layout-container">
     <div class="resource-title">
       <i class="resource-icon resource-org"></i>
-      <h1>{{organization.name}} Billing</h1>
+      <h1>{{billingDetail.organization.name}} Billing</h1>
```

Comment:
You can remove the `isLoaded` check and property from this component entirely. Since the model is loaded in the route, it will block the rendering of this component entirely until loaded, so the `Loading...` indicator is never going to be visible.